R/ecdc.R

```diff
@@ -59,19 +51,17 @@ get_ecdc_cases <- function (countries = NULL){
     
   }
 
-  d <- mem_read(base_url) %>%
+  data <- csv_reader(file = url) %>%
     dplyr::mutate(date = as.Date(dateRep, format = "%d/%m/%Y")) %>%
-    dplyr::rename(geoid = geoId, country = countriesAndTerritories,
-           cases = cases, deaths = deaths,
-           population_2019 = popData2019) %>%
-    dplyr::select(-dateRep) %>%
+    dplyr::rename(iso_code = geoId, country = countriesAndTerritories,
+                  continent = continentExp, cases_new = cases, deaths_new = deaths,
+                  population_2019 = popData2019) %>%
```

Comment:
If this cleaning code is shared between the spread sheet and the csv it should only happen once so its easier to maintain

Reply:
fixed